velox/exec/fuzzer/AggregationFuzzerBase.cpp

```diff
@@ -239,7 +239,13 @@ std::vector<std::string> AggregationFuzzerBase::generateSortingKeys(
     std::vector<std::string>& names,
     std::vector<TypePtr>& types) {
   std::vector<std::string> keys;
-  auto numKeys = boost::random::uniform_int_distribution<uint32_t>(1, 5)(rng_);
+  // 30% chance that there will only be 1 order by key so k-RANGE frames can be
+  // generated
```

Comment:
What about we determine the frame type before calling generateSortingKeys, so that we can determine the number of order-by keys more purposely?

Reply:
Will determine the frame type before calling generateSortingKeys(). And then if frame type == kRange then just generate sorting keys of size, else generate sorting keys of random size (1,5) as in the code